src/pages/en/core-concepts/astro-components.md

```diff
@@ -335,6 +334,9 @@ To send JavaScript to the browser without [using a framework component](/en/core
   // Processed! Bundled! ESM imports work, even to npm packages.
 </script>
 ```
+
+📚 See our [directives reference](/en/reference/directives-reference#script-and-style-tags) page for more information about the directives available  on `<script>` tags.
```

Comment:
1. Is "directives" a standard term that we expect readers to understand? 

In framework-components, we explain directives as "a component attribute to define how your component should be rendered and hydrated." Is there some similar phrase we could add here so that "directive" doesn't just sneak up out of no where? (I'm not sure I would have known what a "directive" was when I started with Astro, but I did know and understand the word "attribute.") 

I checked the entire page and we do mention "special Astro directives" earlier in this page, but also don't explain/define, and maybe we should in at least one place? Here or there? Also note that we don't mention anything about directives in the section directly above, re CSS styles. So maybe we should there, too? This page doesn't indicate that there are any `<style>` directives available.

2. (slightly lower down) This page mentions "hoist" and I think that's going away, if it hasn't already. Probably safe for the purposes of this PR to consider it gone?

Reply:
Quite possibly, but I think changing either of those things are out of this PR's scope.

I think that for 1, the change should be made in the earlier reference in the page.